packages/frontend/src/components/swap/components/helpers.js

```diff
@@ -0,0 +1,223 @@
+import * as nearApiJs from 'near-api-js';
```

Comment:
findTokenSwapToList, exchangeRateTranslation, and useInterval are the only new code added